admin-dev/themes/new-theme/scss/pages/_orders_create.scss

```diff
@@ -0,0 +1,8 @@
+#order-creation-container {
+  .custom-fields-container {
+    max-height: 300px;
+    overflow-y: scroll;
+    padding-bottom: 0.9375rem;
```

Comment:
Simple question, wouldn't be better to us the ps-rail like the left sidebar of BO ? Would looks the same on every computers

Reply:
sorry i dont know about that :disappointed: , haven't done much with front end, so idk whats ps-rail :sweat_smile: . Maybe someone else has answer to that ?

Reply:
That's the scroll bar used in the menu sidebar of the BO, I never used it too but it looks not difficult :p

Reply:
FYI, Worked on the scrollbar of menu, It's perfect scrollbar, that would be awesome to also use it for pages